manifests/capistrano.pp

```diff
@@ -13,6 +13,14 @@
     provider => 'gem',
     notify   => Exec['update_rubygems']
   }
+  -> package { 'redis':
```

Comment:
How is capistrano related to redis? My understanding is not at all? Redis is purely deploynaut dependency, so should probably be added to `deploynaut::install`. And why do we need resque dependency?